src/o365/spo/commands/listitem/listitem-batch-add.spec.ts

```diff
@@ -0,0 +1,400 @@
+import commands from '../../commands';
+import Command from '../../../../Command';
+import { CommandValidate, CommandOption, CommandTypes } from '../../../../Command';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+import auth from '../../../../Auth';
+const command: Command = require('./listitem-add');
```

Comment:
This points to the `spo listitem add` command rather than `spo listitem batch add`

Reply:
ouch!, havent even looked at the tests. they were just copied over from ./listitem-add
any recommendations/links/resources on how i can learn to set these up?

Reply:
I think the easiest way is to look at other commands and use their tests as a starting point. If you come across something specific that you can't find anywhere else, let us know and we'll figure it out together.